pipeline/outputs/datadog.md

```diff
@@ -24,7 +24,8 @@ Before you begin, you need a [Datadog account](https://app.datadoghq.com/signup)
 | dd_source       | _Recommended_ - A human readable name for the underlying technology of your service (e.g. `postgres` or `nginx`). If unset, Datadog will look for the source in the [`ddsource` attribute](https://docs.datadoghq.com/logs/log_configuration/pipelines/?tab=source#source-attribute).                                                                                                                                                                                                                                                                                                                   |                                  |
 | dd_tags         | _Optional_ - The [tags](https://docs.datadoghq.com/tagging/) you want to assign to your logs in Datadog. If unset, Datadog will look for the tags in the [`ddtags' attribute](https://docs.datadoghq.com/api/latest/logs/#send-logs).                                                                                                                                                                                                                                                                                                                                   |                                  |
-| dd_message_key  | By default, the plugin searches for the key 'log' and remap the value to the key 'message'. If the property is set, the plugin will search the property name key.                                                                                                                                                                                                                                                                          |                                  |
+| dd_message_key  | By default, the plugin searches for the key 'log' and remap the value to the key 'message'. If the property is set, the plugin will search the property name key.                                                                                                                                                                                                                                                                                                                                                                                                       |                                  |
+| dd_hostname     | The host that emitted logs should be associated with. If unset, Datadog will expect the host to be set as `host`, `hostname`, or `syslog.hostname` attributes. See Datadog Logs preprocessor documentation for up-to-date recognized attributes.                                                                                                                                                                                                                                                                                                                       |                                  |
```

Comment:
Is there a link we can use to link the "Datadog Logs preprocessor documentation"?

Reply:
👍 added a link in [c4d9263](https://github.com/fluent/fluent-bit-docs/pull/1401/commits/c4d9263d8836e599f7a186acf482b427de16a46a)

Reply:
Nice! Thank you.